indexer.go

```diff
@@ -44,5 +47,44 @@ func (c CombinedIndexer) Get(ctx context.Context, opts *packages.GetOptions) (pa
 		}
 		packages = packages.Join(p)
 	}
+
+	if !opts.Filter.AllVersions {
+		return latestPackagesVersion(packages), nil
+	}
+
 	return packages, nil
 }
+
+func latestPackagesVersion(source packages.Packages) (result packages.Packages) {
+	sort.Sort(byNameVersion(source))
```

Comment:
If you call `sort.Sort` on the source, it won't alter the original `packages` slice? I'm double-checking if this is concurrency-safe.

Reply:
It will modify the `packages` slice defined in the `CombinerIndexer` `Get`, that is built on every call to this method and shouldn't be used concurrently in any other place.